src/test.cpp

```diff
@@ -289,7 +289,7 @@ int main(int argc, char** argv) {
 #else
     const char* mode = "";
 #endif
-    printf("Running libminisketch tests%s with complexity=%llu\n", mode, (unsigned long long)test_complexity);
+    printf("Running libminisketch tests%s with complexity=%lu\n", mode, uint64_t{test_complexity});
```

Comment:
I think if we want this to be `unsigned long long` and not uint64, then I'll need to use a static_cast (since it's a multiple word identifier). Up to you

Reply:
Okay. reverted